apis/sgv2-quarkus-common/src/main/java/io/stargate/sgv2/api/common/config/GrpcConfig.java

```diff
@@ -29,4 +34,25 @@ public interface GrpcConfig {
   /** @return Optional deadline duration for the each RPC to the bridge. Defaults to 30 seconds. */
   @WithDefault("PT30S")
   Optional<Duration> callDeadline();
+
+  /** @return Defines retry strategy for bridge calls when using {@link RetriableStargateBridge}. */
+  @Valid
+  @NotNull
+  Retries retries();
+
+  interface Retries {
+
+    /** @return If call retries are enabled. */
+    @WithDefault("true")
+    boolean enabled();
+
+    /** @return List of status codes to execute retries for. */
```

Comment:
Maybe add a comment on why we include, by default, UNAVAILABLE? (so it's easy to see in a few months when looking at this code).